src/lib-components/SectionStaffSubjectLibrarian.vue

```diff
@@ -0,0 +1,117 @@
+<script
+  setup
+  lang="ts"
+>
+import type { PropType } from 'vue'
+
+// TYPESCRIPT
+import type { BlockStaffListItemType } from '@/types/types'
+
+import BlockStaffSubjectLibrarian from '@/lib-components/BlockStaffSubjectLibrarian.vue'
+
+const { items, tableHeaders } = defineProps({
+  items: {
+    type: Array as PropType<BlockStaffListItemType[]>,
+    default: () => [],
+  },
+  tableHeaders: {
+    type: Array,
+    default: () => [],
+  },
+})
+</script>
+
+<template>
+  <table class="section-staff-subject-librarian">
+    <caption>
+      Subject Librarians
+    </caption>
+    <thead>
+      <tr>
+        <th
+          v-for="(header, index) in tableHeaders"
+          :key="index"
+        >
+          {{ header }}
+        </th>
+      </tr>
+    </thead>
+
+    <tbody>
+      <BlockStaffSubjectLibrarian
+        v-for="(item, index) in items"
+        :key="`${index}-${item.subjectArea}`"
+        :subject-area="item.subjectArea"
+        :staff-name="item.staffName"
```

Comment:
I think something is wrong here or somewhere with the name in SectionStaffSubjectionLibrarian. It shows okay now in the block, but in the section view it only shows the alternative name, regular first and last name are missing
<img width="1057" alt="Screenshot 2024-04-09 at 12 27 55 PM" src="https://github.com/UCLALibrary/ucla-library-website-components/assets/7073987/0647b931-d7e0-46b3-8674-07644001148e">

Reply:
Refactored this.